src/ToolBox/SOS/NETCore/project.json

```diff
@@ -11,14 +11,5 @@
        "portable-net45+win8" 
       ] 
     } 
-  },
```

Comment:
Does this produce the expected artifacts for both portable and non-portable builds (and do they work)?

Do "build.cmd -portable" for triggering a portable build.